python/cudf_polars/tests/expressions/test_numeric_unaryops.py

```diff
@@ -24,8 +25,12 @@
         "arcsinh",
         "arccosh",
         "arctanh",
-        # "exp", Missing rust side impl
-        # "log", Missing rust side impl
+        pytest.param(
```

Comment:
If exp and log are not yet supported in 1.5, what's the benefit of bumping the Polars version in this PR?

Reply:
I can just bump straight to 1.6

Reply:
Either is fine. Whatever you want to put in this PR vs future ones for the version bump is OK with me.

Reply:
Updated here (and culled a few "compat" bits that are now unnecessary).